### PR TITLE
chore(print): remove pagedjs + prepare for alpha

### PR DESCRIPTION
**Pull request by usu**
Based on #3024 

- [x] remove pagedjs from nuxt print
- [x] implement simple screen preview for nuxt print
- [x] hide developer previews
- [x] refine user experience to select a print engine 

# Before
image

# After
image

User view:
image

Developer view (on localhost):
image

### CODE REVIEW

File: frontend/src/locales/de.json [hunk 1]
```diff
@@ -127,7 +127,7 @@
       },
       "printReact": {
         "downloadNuxtPdfListItem": {
```
Comment (BacLuc) [inline frontend/src/locales/de.json]: wieso heissen hier beide nuxt?

File: frontend/src/locales/en.json [hunk 2]
```diff
@@ -138,6 +138,16 @@
       },
       "printPreviewNuxt": {
         "openPreview": "Open preview in new window"
+      },
+      "printReact": {
```
Comment (BacLuc) [inline frontend/src/locales/en.json]: i would keep the same order as in de.json

File: frontend/src/components/print/print-react/DownloadReactPdfListItem.vue [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <v-list-item :disabled="loading" @click="generatePdf">
     <v-list-item-icon>
       <v-icon v-if="loading" class="mdi-spin">mdi-loading</v-icon>
-      <v-icon v-else>mdi-react</v-icon>
+      <v-icon v-else>mdi-printer</v-icon>
     </v-list-item-icon>
     <v-list-item-title>
       {{ $tc('components.print.printReact.downloadNuxtPdfListItem.label') }}
```
Comment (pmattmann) [inline frontend/src/components/print/print-react/DownloadReactPdfListItem.vue]: components.print.printReact.download _Nuxt_ PdfListItem.label
rename to
components.print.printReact.downloadReactPdfListItem.label

**Comment by pmattmann** [comment]
nice

File: .helm/ecamp3/templates/print_configmap.yaml [hunk 4]
```diff
@@ -7,3 +7,5 @@ metadata:
 data:
   INTERNAL_API_ROOT_URL: {{ include "api.url" . | quote }}
   FRONTEND_URL: {{ include "frontend.url" . | quote }}
+  PRINT_SERVER: {{ include "print.url" . | quote }}
```
Comment (carlobeltrame) [inline .helm/ecamp3/templates/print_configmap.yaml]: Any reason why this isn't consistently called `PRINT_URL`?

File: frontend/src/views/camp/Story.vue [hunk 5]
```diff
@@ -5,38 +5,22 @@ Admin screen of a camp: Displays details & periods of a single camp and allows t
...
-        <e-switch
-          v-model="editing"
-          :disabled="!isContributor"
-          :label="$tc('global.button.editable')"
-          class="ec-story-editable ml-auto"
-          @click="$event.preventDefault()"
-        />
-      </template>
-      <v-menu v-else offset-y>
+      <LockIcon v-model="editing" :hide-tooltip="isContributor" />
```
Comment (carlobeltrame) [inline frontend/src/views/camp/Story.vue:6]: Please use kebab-case for vue components: https://v2.vuejs.org/v2/style-guide/?redirect=true#Component-name-casing-in-templates-strongly-recommended

File: print/print.env [hunk 6]
```diff
@@ -2,3 +2,5 @@ INTERNAL_API_ROOT_URL=http://caddy:3001
 FRONTEND_URL=http://localhost:3000
 SENTRY_PRINT_DSN=
 BROWSER_WS_ENDPOINT=ws://browserless:3000
+PRINT_SERVER=http://print:3003
```
Comment (carlobeltrame) [inline print/print.env]: Also here, PRINT_URL?

**Review by carlobeltrame — APPROVED** [review]
I'm still convinced the generic-error-message could be done better. But I can open an issue for that and do it some time in the future.

File: print/print.env (hunk 6 above)
Comment (usu) [reply print/print.env]: Done in https://github.com/ecamp/ecamp3/pull/3036/commits/483415521cf70f767b7e75b9b6d3c4c65df3d741

File: .helm/ecamp3/templates/print_configmap.yaml (hunk 4 above)
Comment (usu) [reply .helm/ecamp3/templates/print_configmap.yaml]: Done in https://github.com/ecamp/ecamp3/pull/3036/commits/483415521cf70f767b7e75b9b6d3c4c65df3d741

File: frontend/src/components/print/print-react/DownloadReactPdfListItem.vue (hunk 3 above)
Comment (usu) [reply frontend/src/components/print/print-react/DownloadReactPdfListItem.vue]: Fixed in https://github.com/ecamp/ecamp3/pull/3036/commits/abe5bd69b9ce833ec7f3432f27a82c851004e91e

File: frontend/src/locales/en.json (hunk 2 above)
Comment (usu) [reply frontend/src/locales/en.json]: Fixed in https://github.com/ecamp/ecamp3/pull/3036/commits/abe5bd69b9ce833ec7f3432f27a82c851004e91e

File: frontend/src/locales/de.json (hunk 1 above)
Comment (usu) [reply frontend/src/locales/de.json]: Fixed in https://github.com/ecamp/ecamp3/pull/3036/commits/abe5bd69b9ce833ec7f3432f27a82c851004e91e

File: frontend/src/views/camp/Story.vue (hunk 5 above)
Comment (carlobeltrame) [reply frontend/src/views/camp/Story.vue:6]: Nevermind, let's talk about that some other time in #695

File: frontend/src/components/print/print-nuxt/DownloadNuxtPdfButton.vue [hunk 7]
```diff
@@ -2,8 +2,9 @@
   <div>
     <v-btn color="primary" :loading="loading" outlined @click="generatePdf">
```
Comment (carlobeltrame) [inline frontend/src/components/print/print-nuxt/DownloadNuxtPdfButton.vue:3]:
```suggestion
    <v-btn color="primary" :loading="loading" outlined @click.stop="generatePdf">
```

File: frontend/src/components/print/print-react/DownloadReactPdfButton.vue [hunk 8]
```diff
@@ -1,9 +1,10 @@
 <template>
   <div>
-    <v-btn color="primary" :loading="loading" outlined @click="generatePdf">
+    <v-btn color="primary" :loading="loading" outlined @click.stop="generatePdf">
```
Comment (carlobeltrame) [inline frontend/src/components/print/print-react/DownloadReactPdfButton.vue]:
```suggestion
    <v-btn color="primary" :loading="loading" outlined @click="generatePdf">
```

File: frontend/src/components/print/print-react/DownloadReactPdfListItem.vue [hunk 9]
```diff
@@ -2,70 +2,20 @@
   <v-list-item :disabled="loading" @click="generatePdf">
```
Comment (carlobeltrame) [inline frontend/src/components/print/print-react/DownloadReactPdfListItem.vue]:
```suggestion
  <v-list-item :disabled="loading" @click.stop="generatePdf">
```

**Review by carlobeltrame — APPROVED** [review]
Glaube du hast das `.stop` auf den falschen React Button gemacht

**Review by pmattmann — APPROVED** [review]

**Pull request merged**